### PR TITLE
Implement UI all txs action rows for messages

### LINKED ISSUE #952: Implement UI all txs action rows for messages in #949

### LINKED ISSUE #1021: Fix message send UI bug

### PR DESCRIPTION
**Pull request by Benzbeeb**
Fixed #952 
Fixed #1021

### CODE REVIEW

**Comment by evilpeach** [comment]
@Benzbeeb Please fix the conflict.

**Review by prin-r — CHANGES_REQUESTED** [review]
Screen Shot 2563-03-30 at 09 56 48

Some row didn't align. Can we fix this ?

File: scan/src/components/msgs/Msg.re [hunk 1]
```diff
@@ -21,9 +21,7 @@ let make = (~msg: TxHook.Msg.t, ~width: int, ~success: bool) => {
   switch (msg.action) {
   | Send({fromAddress, toAddress, amount}) =>
     <div className={Styles.rowWithWidth(width)}>
-      <div className={Styles.withWidth(width / 2 - 20)}>
```
Comment (evilpeach) [inline scan/src/components/msgs/Msg.re:24]: Talk to @prin-r. Is it proper to edit this?

**Comment by evilpeach** [comment]
@Benzbeeb I'm going to take care of this PR.

**Review by prin-r — APPROVED** [review]

**Review by evilpeach — APPROVED** [review]

**Pull request merged**